### PR TITLE
Add Hangzhou meetup summary to blog

### PR DESCRIPTION
**Pull request by caojiele**
## What is the purpose of the change

#237 Add Hangzhou meetup summary to blog, But I need someone to help me write down the meeting described [here](https://github.com/caojiele/incubator-dubbo-website/blob/patch-1/blog/zh-cn/dubbo-meetup-hangzhou.md#L10-L11).

## Brief changelog

`modified: blog/en-us/dubbo-meetup-hangzhou.md`
`modified: blog/zh-cn/dubbo-meetup-hangzhou.md`
`modified: img/blog/dubbo-hangzhou-meetup.jpg`
`modified: site_config/community.jsx`

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/incubator-dubbo-website/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Test your code locally by running `docsite start`, and make sure it works as expected.
- [ ] Make sure no files under build directory is added.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/incubator-dubbo/wiki/Software-donation-guide).

### CODE REVIEW

File: blog/en-us/dubbo-meetup-hangzhou.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+keywords: Dubbo, Hangzhou, meetup
+description: The fifth Dubbo meetup has been held in Hangzhou
+---
+
+# The fifth Dubbo meetup has been held in Hangzhou
+
+
+The fifth Dubbo meetup has been held in Hangzhou,
+
+分享嘉宾
```
Comment (lixiaojiee) [inline blog/en-us/dubbo-meetup-hangzhou.md]: Please translate this sentence into English

**Comment by kezhenxu94** [comment]
@caojiele can you please also add an entry to the `site_config/blog.js` file?

**Comment by caojiele** [comment]
> @caojiele can you please also add an entry to the `site_config/blog.js` file?

Are you sure add an entry to `sit_config/blog.js` file?  add to 'sit_config/community.jsx` file?

**Comment by kezhenxu94** [comment]
> > @caojiele can you please also add an entry to the `site_config/blog.js` file?
> 
> Are you sure add an entry to `sit_config/blog.js` file? add to 'sit_config/community.jsx` file?

@caojiele Sorry my fault, add it to `site_config/community.jsx` please

**Comment by caojiele** [comment]
> > > @caojiele can you please also add an entry to the `site_config/blog.js` file?
> > 
> > 
> > Are you sure add an entry to `sit_config/blog.js` file? add to 'sit_config/community.jsx` file?
> 
> @caojiele Sorry my fault, add it to `site_config/community.jsx` please

OK, no problem. Then I changed together in this Pre.

**Comment by kezhenxu94** [comment]
@caojiele Thanks for your contribution

**Comment by caojiele** [comment]
You're welcome！


原始邮件
发件人:kezhenxu94notifications@github.com
收件人:apache/incubator-dubbo-websiteincubator-dubbo-website@noreply.github.com
抄送:Jlcaocaojiele1225@126.com; Mentionmention@noreply.github.com
发送时间:2019年3月8日(周五) 12:55
主题:Re: [apache/incubator-dubbo-website] Add Hangzhou meetup summary toblog (#319)


@caojiele Thanks for your contribution

**Pull request merged**